### ISSUE TITLE
Remove characters function not working on files with mixed Arabic and Latin characters

### ISSUE THREAD

**Issue by queer-bird** [Nomen-Luni/Bionic-Batch-Renamer#13]
[Removed by nomen luni - see below]

**Comment by Nomen-Luni** [comment]
Hi,

If you want to re-post your bug report again minus the political content in the guise of "example strings," feel free.

What is happening in Gaza/Israel is a terrible sadness for all humanity. My sympathies are with the innocent people on both sides who are being hurt.

That's my FINAL comment. Anyone making further political comments regardless of what 'side' they are on will not be allowed to comment here.

I'm giving away free software (to citizens of all nationalities), not running a political discussion forum.

nomen

**Comment by queer-bird** [comment]
I would send this to you privately, but honestly I am not so familiar with Github and to me it seems that that is not a option. Feel free to delete after reading.

My sincerest apologies, I genuinely was not making an attempt at having a political discussion. These were just the actual files in question and I didn't want to leave out any information that might be important for troubleshooting. I'm no expert on this sort of this so I don't know how important the actual exact content of the strings are for such issues. I understand history can be controversial but since these are the files that are giving me the problem, I had no other examples to give. I simply did not put much thought into it and I didn't think anyone else would either. I will ask the same question with more generic strings and put more thought into such things in the future.

I do not want to be confrontational as I came to you and asked you for your precious time, but I would like to offer a bit of food for thought. My family is Jewish so it is not like I have no skin in the game, but I work as an archivist so I must put aside my biases and preserve human knowledge regardless of my opinion on the material in question.

Afghanistan has nothing to with the present conflict and Uyghurs aren't even part of the Arab world. Singling out Arabic script or the history of nations who use it "Poltical" is very questionable. I'm sure these peoples and cultures wouldn't be so thrilled about being lumped together. I doubt it would have been called political if I were having problems with Greek or Ethiopian script/history, despite these things always being inescapably political (as most things involving humans are). There is of course the context of these files being dealt with being patriotic/revolutionary in nature, but again, I feel like if these were songs about the French or American revolution no fuss would have been made despite the fact that they would be political and no doubt offensive to somebody. I'm making no judgements but it is perhaps something worth reflecting on.
